src/specifications/profile-resolution/profile-resolution-specml.xml

```diff
@@ -289,9 +289,20 @@ profile:
       <section id="internalref">
         <head>Internal References</head>
         <p>URI Fragments in OSCAL represent internal references to other OSCAL objects in the same document. These references follow the pattern of <code>#{{objectID}}</code>. For example, the URI Fragment <code>#param1</code> is referencing the Parameter with unique ID <code>param1</code>.</p>
-        <p><req level="must" id="req-internal">In the context of the Import Phase, internal references will only appear as a reference to a profile or catalog that is in the <src>resources</src> section of the source. When tools encounter such a reference, they MUST locate the object in <src>resources</src> with the matching ID value, and resolve the import using the
-          <src>rlink</src> URI and the above resolution requirements.
+        <p><req level="must" id="req-internal">In the context of the Import Phase, internal references will only appear as a reference to a profile or catalog that is in the <src>resources</src> section of the source. When tools encounter such a reference, they MUST locate the object in <src>resources</src> with the matching ID value, and resolve the resource. 
+          By OSCAL model requirements, a given <src>resource</src> can have zero to many <src>rlink</src> objects and zero to one <src>base64</src> objects; however, it must have at least one of the two present. Tools can assume that any of these each resolves to the same underlying OSCAL object, although potentially in different serialization formats.
+          For deterministic resolution of these backmatter OSCAL objects, tools need to adhere to the following requirements:
         </req></p>
+        <ul>
+          <li><p><req level="may" id="req-internal-resolve1">Tools MAY use any of the <src>rlink</src> or <src>base64</src> objects present in the <src>resource</src>.</req> </p> </li>
+          <li><p><req level="may" id="req-internal-resolve2">Tools MAY verify that each above objects resolves to the same underlying OSCAL object. If there are OSCAL Model content differences between the resolved obejcts, the tool SHOULD provide a warning. </req></p> </li>
+          <li><p><req level="must" id="req-internal-resolve3">When a <src>rlink</src> is encountered and is to be resolved, it MUST be resolved by using a HTTP Client request to retrieve a Byte Stream. </req></p> </li>
```

Comment:
Does HTTP Client request mean something specific from a RFC or standard? I ask because it is capitalized. I assume we do not want to be so prescriptive as to say HTTP 1.1 or HTTP 2, or the kind of request (`GET`, `POST`, et cetera), but I was thrown off by the capitalization. Should we keep it sufficiently generic and just say HTTP request?

Similarly, I have only seen "byte stream" written with capitals ("Byte Stream") when discussing specific programming languages that implement classes with names like `ByteStream` or such. Do we want to capitalize it?

I only point these things out because to me capitals in many contexts means: specific noun, which is likely terminology that I (as an outside developer) will need to look up to make sure I am not missing something I thought I knew. My $0.02, since this is stylistic, definitely take it or leave it kind of AJ suggestions here.

```suggestion
          <li><p><req level="must" id="req-internal-resolve3">When a <src>rlink</src> is encountered and is to be resolved, it MUST be resolved by using a HTTP request to retrieve a byte stream. </req></p> </li>
```